app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# Users controller
+class UsersController < ApplicationController
+  before_action :require_login
```

Comment:
This now runs for all methods in this controller, instead of just `show`, `past` and `future`, as was the case before. Include an `only` param to fix this.